person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
Both attribute accessors and readers should be mentioned at the top just after declaring the class. Remember, to always practice Ruby's best practices.

Reply:
Implemented 👍